libindi/drivers/telescope/telescope_script.cpp

```diff
@@ -79,7 +78,11 @@ const char * ScopeScript::getDefaultName() {
 bool ScopeScript::initProperties() {
   INDI::Telescope::initProperties();
   
+#ifdef OSX_EMBEDED_MODE
   IUFillText(&ScriptsT[0], "FOLDER", "Folder", "/usr/local/share/indi/scripts");
+#else
+  IUFillText(&ScriptsT[0], "FOLDER", "Folder", "/usr/share/indi/scripts");
```

Comment:
Regarding the location as well, INDI won't be shipping any scripts, they're always user-provided. So in reality maybe it doesn't matter where they end up, and the user can change the folder anyway.